compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/SyntaxErrors.java

```diff
@@ -240,6 +240,8 @@ private static DiagnosticCode getErrorCode(SyntaxKind expectedKind) {
                 return DiagnosticErrorCode.ERROR_MISSING_JOIN_KEYWORD;
             case OUTER_KEYWORD:
                 return DiagnosticErrorCode.ERROR_MISSING_OUTER_KEYWORD;
+            case CLASS_KEYWORD:
+                return DiagnosticErrorCode.ERROR_MISSING_CLASS_KEYWORD;
```

Comment:
`getErrorCode(Syntaxkind expectedKind)` will be removed soon. There is a new method introduced `getErrorCode(ParserRuleContext currentCtx)` Please add all new contexts which u introduce to that method as u add in getExpectedToken in ErrorHandler.

Reply:
I see. class-change branch haven't synced with this change from master it seems. Will fix once synced. Thanks!

Reply:
Cool

Reply:
@rdulmina Shall we remove that `getErrorCode(Syntaxkind expectedKind)` soon?

Reply:
@SupunS sure. Here is the issue created https://github.com/ballerina-platform/ballerina-lang/issues/25611

Reply:
@rdulmina added to `getErrorCode(ParserRuleContext currentCtx)` with b3b7d5b in PR #25623

Reply:
awesom